cxx-sensors/src/tools/cppcheck_createrules.py

```diff
@@ -26,10 +26,14 @@
 import sys
 import xml.etree.ElementTree as et
 
-severity_2_priority = {
-    "error": "MAJOR",
-    "style": "MINOR"
-    }
+severity_2_sonarqube = {
```

Comment:
not sure about this mapping too, but for the new rules we have to double-check the type/priority manually in any way